src/panels/lovelace/common/evaluate-filter.ts

```diff
@@ -1,12 +1,32 @@
 import { HassEntity } from "home-assistant-js-websocket";
+import { HomeAssistant } from "../../../types";
+import { isValidEntityId } from "../../../common/entity/valid_entity_id";
+
+export const evaluateFilter = (
+  hass: HomeAssistant,
+  current_entity_id: string,
+  filter: any
```

Comment:
```ts
filter: FilterDeprecated | Filter

FilterDeprecated {
  operator: string;
  attribute?: string;
  state: string | number;
}

Filter {
  condition: "state" | "numeric_state";
  attribute?: string;
  state?: string | number;
  below?: string | number;
  above?: string | number;
}
```

Reply:
Hmm, the deprecated operators were:
`==`, `<=`, `<`, `>=`, `>`, `!=`, `in`, `not in`, or `regex`.

I don't think there's a replacment for a lot of these in the new syntax?

Reply:
> I don't think there's a replacment for a lot of these in the new syntax?

Correct, they will stay in the old format only

Reply:
While developing 8c929545174c7b907cd28065ba4817696094b991, I typed for `condition: numeric_state`, `condition: state` and `condition: operator`, which is the old syntax, that still works property